### DISCUSSION TITLE
Google Project IDX

### DISCUSSION THREAD

**Discussion by ldijkman** [ldijkman/Ace_Seventh_Heaven#4] [Show and tell]
What is Google IDX?

Google's Project IDX is a new AI-powered IDE (integrated development environment) 
that aims to make full-stack app development easier and more accessible. 
Project IDX is built on Google Cloud and powered by Codey, a foundational AI model trained on code.

webpage looks a bit like prettier logo

like when Google anounced Android, when was it around 2000
 
https://idx.dev/

**Comment by Jaomar92** [comment]
Hi, i figured you have toyed around with the IDE already. But How do I push the code i have made on IDX to my github repo? From my PC it works but from a VM? I am not sure how it works.
